### PR TITLE
IS Gear, and other fixes.

### PR DESCRIPTION
**Pull request by Iankid**
-Added gear from IS.
-Added rules referenced.
-Added Hydrodynamic weapon property
-Added missing feats from IS
-Removed "projectile" weapon catagory from Frag grenades.
-Changed Stellar mode modifiers to situational

### CODE REVIEW

File: src/items/equipment/panic_grenade_mk_1.json [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+    "quantityPerPack": 1,
+    "range": {
+      "additional": "",
+      "per": "",
+      "units": "ft",
+      "value": 20
+    },
+    "rollNotes": "",
+    "save": {
+      "type": "reflex",
```
Comment (LebombJames) [inline src/items/equipment/panic_grenade_mk_1.json:211]: The description calls for a will save, not a reflex save. Same for all other marks

File: src/items/equipment/psychic_strike_node_advanced.json [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    },
+    "equippable": true,
+    "equipped": false,
+    "identified": true,
+    "level": 14,
+    "modifiers": [
+      {
+        "_id": "7b217b75-2573-4191-a101-d50ef3cc5f64",
+        "name": "Deception node",
```
Comment (LebombJames) [inline src/items/equipment/psychic_strike_node_advanced.json:66]: Looks like an incorrect duplication. Can add a damage entry too. The other marks need fixing too.

File: src/items/equipment/singing_stinger.json [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+          "affectsEncumbrance": true,
+          "amount": 1,
+          "subtype": "ammunitionSlot",
+          "weightMultiplier": 1,
+          "weightProperty": ""
+        }
+      ]
+    },
+    "critical": {
+      "effect": "Injection +2",
```
Comment (LebombJames) [inline src/items/equipment/singing_stinger.json:90]: Should be "Injection DC +2"

File: src/items/equipment/spell_amulet_1st.json [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+        "source": "",
+        "subtab": "misc",
+        "valueAffected": "both"
+      },
+      {
+        "_id": "da95213a-bc11-4189-a42a-2e4963b4e9fc",
+        "name": "Spell amulet: Divination +1",
+        "type": "circumstance",
+        "condition": "",
+        "effectType": "saves",
```
Comment (LebombJames) [inline src/items/equipment/spell_amulet_1st.json:106]: The saves can be situational, I assume the other levels also need to be changed

File: src/items/equipment/spell_amulet_1st.json [hunk 5]
```diff
@@ -0,0 +1,171 @@
+{
+  "_id": "vdRJy2t9VydjsxHM",
+  "name": "Spell amulet 1st",
```
Comment (LebombJames) [inline src/items/equipment/spell_amulet_1st.json:3]: The naming scheme should match Spell Gems, which have a comma before the level iirc (e.g "Spell Amulet, 1st")

File: src/items/equipment/valai_band.json [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
...pendium.sfrpg.races.6qGAoeERMRvuQjUw]{vlakas} during valai season, this metal band is traditionally worn around the neck in an echo of the vlakan Leader’s Band, but can also be worn on an arm, leg, or similar appendage. When activated, it provides environmental protection within a 10-foot radius of the wearer that is equivalent to wearing armor (Core Rulebook 196). It may also, at the wearer’s discretion, emit a homing signal every 10 seconds for 24 hours at the cost of additional charge. The homing signal can be picked up by most devices capable of receiving signals and tracked back to the source with a successful DC 15 @UUID[Compendium.sfrpg.rules.GMkLZsN3a7YPvA03.JournalEntryPage.76u4HnGcAlySicb4]{Engineering} or @UUID[Compendium.sfrpg.rules.GMkLZsN3a7YPvA03.JournalEntryPage.UDTsEIldhXTn0VJA]{Survival} check. A single valai band’s homing signal has a range of 100 miles over flat terrain (halved in forests or hills and quartered in mountainous terrain or underwater). If multiple valai bands are present within the same 10-foot radius, the homing range increases by a factor of 2 for each additional band, to a maximum of a planetary range.</p><p><strong>Usage</strong> 1/8 Hours</p>"
```
Comment (LebombJames) [inline src/items/equipment/valai_band.json:62]: What's with the Usage entry in the description?

File: src/items/equipment/vali_water.json [hunk 7]
```diff
@@ -0,0 +1,190 @@
...
+      "units": "",
+      "value": null
+    },
+    "equippable": false,
+    "equipped": false,
+    "formula": "",
+    "identified": true,
+    "isActive": null,
+    "level": 16,
+    "modifiers": [
```
Comment (LebombJames) [inline src/items/equipment/vali_water.json:100]: Do modifiers work like this on consumables? Or do they need to be active?

**Review by LebombJames — CHANGES_REQUESTED** [review]
I haven't checked the new journals in foundry yet

File: src/items/equipment/valai_band.json (hunk 6 above)
Comment (Iankid) [reply src/items/equipment/valai_band.json:62]: Usage is 1 every 8 hours. we do not have that as an option.

File: src/items/equipment/vali_water.json (hunk 7 above)
Comment (Iankid) [reply src/items/equipment/vali_water.json:100]: There is no activation button in sheet for consumables. So unless there's a better solution present, or one made. the consumable needs modifiers manually turned on/off

File: src/items/equipment/psychic_strike_node_advanced.json (hunk 2 above)
Comment (Iankid) [reply src/items/equipment/psychic_strike_node_advanced.json:66]: Not sure what you mean by incorrect duplication. This augment can be gained twice with a few races, and the modifier is different for two augments. Perhaps a name change.
As for damage. Augments currently do not have a damage function.

File: src/items/equipment/valai_band.json (hunk 6 above)
Comment (Iankid) [reply src/items/equipment/valai_band.json]: Had to add 8 hours to the UsagePer options to complete this one.

File: src/items/equipment/psychic_strike_node_advanced.json (hunk 2 above)
Comment (LebombJames) [reply src/items/equipment/psychic_strike_node_advanced.json:66]: This modifier seems to have no relevance to this item, and seems to be for an entirely different item

File: src/items/rules/movement_and_position.json [hunk 8]
```diff
@@ -0,0 +1,274 @@
...
...(all temperatures)</td><td>×1</td><td>×1/2</td><td>×1/2</td></tr><tr><td>Forest (typical)</td><td>×1</td><td>×1</td><td>×1/2</td></tr><tr><td>Forest (dense)</td><td>×1</td><td>×3/4</td><td>×1/4</td></tr><tr><td>Hill</td><td>×1</td><td>×3/4</td><td>×1/2</td></tr><tr><td>Marsh</td><td>×1</td><td>×3/4</td><td>×1/2</td></tr><tr><td>Mountain</td><td>×3/4</td><td>×3/ 4</td><td>×1/2</td></tr><tr><td>Plains</td><td>×1</td><td>×1</td><td>×3/4</td></tr><tr><td>Urban</td><td>×3/4</td><td>×1/2</td><td>×1/2</td></tr></tbody></table><h2>Forced March</h2><p>In a day of normal walking, you walk for 8 hours, and then you spend the rest of your daylight time making and breaking camp, resting, and eating. However, you can walk for more than 8 hours in a day by making a forced march. For each hour of marching beyond 8 hours, you must succeed at a Constitution check (DC = 10 + 2 per extra hour) or you take 1d6 nonlethal damage. If you take any nonlethal damage from a forced march, you become fatigued (see page 276). When you recover from this nonlethal damage, you also eliminate the fatigued condition. Still, it’s quite possible to march yourself into unconsciousness by pushing yourself too hard.</p>",
```
Comment (LebombJames) [inline src/items/rules/movement_and_position.json:208]: Link to fatigued under forced march

**Review by LebombJames — CHANGES_REQUESTED** [review]

**Pull request merged**